[Do Not Merge] Add LXD container build scripts for ACA

VanderChen opened the pull request.

cj-chung commented:
@VanderChen I think remain this PR in ACA's repository, but please make a new folder (e.g., lxd-container) under build folder, them put all your changes to this folder. Thanks.

VanderChen commented:
> @VanderChen I think let's remain this PR in ACA's repository, but please make a new folder (e.g., lxd-container) under build folder, them put all your changes to this folder. Thanks.

@cj-chung I have moved the scripts into lxd-container folder and added a  function for installing distrobuilder.
